File: articles/authority-power-institutional-leadership/fortran/authority_governance_score.f90
! Authority and Power in Institutional Leadership
! Simple Fortran scoring demonstration.
!
! Responsible-use scope:
! Synthetic-data research, methods demonstration, institutional learning,
! and reproducible workflows only. Not for employment, productivity ranking,
! loyalty scoring, political-influence scoring, dissent tracking, or individual
! decisions.

program authority_governance_score_program
  implicit none
  real :: formal_legitimacy, procedural_fairness, informational_access
  real :: practical_influence, ethical_credibility, accountability_quality
  real :: voice_access, oversight_strength, arbitrariness_risk
  real :: governance_fragmentation, distrust_pressure
  real :: informal_power_concentration, score

  formal_legitimacy = 84.0
  procedural_fairness = 82.0
  informational_access = 79.0
  practical_influence = 81.0
  ethical_credibility = 83.0
  accountability_quality = 82.0
  voice_access = 80.0
  oversight_strength = 78.0
  arbitrariness_risk = 18.0
  governance_fragmentation = 24.0
  distrust_pressure = 20.0
  informal_power_concentration = 22.0

  score = 0.12 * formal_legitimacy + 0.12 * procedural_fairness + &
          0.10 * informational_access + 0.10 * practical_influence + &
          0.12 * ethical_credibility + 0.11 * accountability_quality + &
          0.10 * voice_access + 0.09 * oversight_strength - &
          0.08 * arbitrariness_risk - 0.07 * governance_fragmentation - &
          0.07 * distrust_pressure - 0.05 * informal_power_concentration

  print *, "Synthetic authority effectiveness score:", score
  print *, "Responsible-use reminder: institutional learning only."
end program authority_governance_score_program

File: articles/authority-power-institutional-leadership/go/authority_governance_score.go
// Authority and Power in Institutional Leadership
// Simple Go scoring demonstration.
//
// Responsible-use scope:
// Synthetic-data research, methods demonstration, institutional learning,
// and reproducible workflows only. Not for employment, productivity ranking,
// loyalty scoring, political-influence scoring, dissent tracking, or individual
// decisions.

package main

import "fmt"

type AuthorityGovernanceProfile struct {
	FormalLegitimacy          float64
	ProceduralFairness        float64
	InformationalAccess       float64
	PracticalInfluence        float64
	EthicalCredibility        float64
	AccountabilityQuality     float64
	VoiceAccess              float64
	OversightStrength         float64
	ArbitrarinessRisk         float64
	GovernanceFragmentation   float64
	DistrustPressure          float64
	InformalPowerConcentration float64
}

func AuthorityEffectivenessScore(p AuthorityGovernanceProfile) float64 {
	return 0.12*p.FormalLegitimacy +
		0.12*p.ProceduralFairness +
		0.10*p.InformationalAccess +
		0.10*p.PracticalInfluence +
		0.12*p.EthicalCredibility +
		0.11*p.AccountabilityQuality +
		0.10*p.VoiceAccess +
		0.09*p.OversightStrength -
		0.08*p.ArbitrarinessRisk -
		0.07*p.GovernanceFragmentation -
		0.07*p.DistrustPressure -
		0.05*p.InformalPowerConcentration
}

func main() {
	profile := AuthorityGovernanceProfile{84, 82, 79, 81, 83, 82, 80, 78, 18, 24, 20, 22}
	fmt.Printf("Synthetic authority effectiveness score: %.2f\n", AuthorityEffectivenessScore(profile))
	fmt.Println("Responsible-use reminder: institutional learning only.")
}
